src/styles/utils/_mixin.scss

```
@@ -0,0 +1,5 @@
+@mixin pos {
```

```suggestion
@mixin position {
```
`pos` is very strange name